pkg/assembler/graphql/schema/package.graphql

```diff
@@ -126,6 +130,7 @@ on nodes that don't contain any qualifier, set `matchOnlyEmptyQualifiers` to
 true. If this field is true, then the qualifiers argument is ignored.
 """
 input PkgSpec {
+  id: ID
```

Comment:
I thought IDs were only needed for evidence trees.. i guess it doesn't hurt.

Reply:
i guess there's a use case for https://github.com/guacsec/guac/pull/593#discussion_r1138676596